package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
```

Comment:
- [ ] Inorder to run and open the server you should add the following script 👇 
```json
        "start": "webpack serve --open",
```

Reply:
Hi @hiromataba,
Thanks for your review!
I'm using the dev script to lunch the server, is not the same?
Thanks in advance for your reply